tests/functional/reporting-non-auth.js

```diff
@@ -267,39 +266,21 @@ registerSuite("Reporting (non-auth)", {
         this,
         url("/issues/new?details=" + DETAILS_STRING),
         "#description"
-      )
-        .findByCssSelector("#steps_reproduce")
-        .getProperty("value")
-        .then(function(text) {
-          assert.include(
-            text,
-            "Encountered error: NS_ERROR_DOM_MEDIA_DEMUXER_ERR (0x806e000c)\nLocation: virtual\nRefPtrMP4Demuxer::InitPromise mozilla::MP4Demuxer::Init()\nError information:\nIncomplete MP4 metadata\nMedia URL: file:///Users/potch/Documents/mozilla/media.mp4"
-          );
-        })
-        .end();
-    },
-
-    "details param: + decoded as space"() {
-      return FunctionalHelpers.openPage(
-        this,
-        url("/issues/new?details=" + DETAILS_STRING2),
-        "#description"
       )
         .findByCssSelector("#steps_reproduce")
         .getProperty("value")
         .then(function(text) {
           assert.notInclude(
             text,
-            "Encountered+error:+NS_ERROR_DOM_MEDIA_DEMUXER_ERR",
+            "gfx.webrender.all:+false",
             "+ not found in decoded string"
           );
           assert.include(
```

Comment:
maybe an assert.include with two lines instead of only one. so we can test that there hard return worked.